README.md

```diff
@@ -115,20 +112,18 @@ func main() {
 	seqset, _ := imap.NewSeqSet("")
 	seqset.AddRange(mbox.Messages - 3, mbox.Messages)
 
-	messages := make(chan *imap.Message)
-	done = make(chan error, 1)
+	messages := make(chan *imap.Message, 10)
```

Comment:
I didn't realized when I first looked that you used a buffered channel here. Is there a particular reason?

The `for msg := range messages` loop should process messages as soon as they arrive.